request_test.ts

```diff
@@ -2,13 +2,13 @@
 
 import {
   test,
+  assert,
   assertEquals,
   assertStrictEq,
+  decoder,
```

Comment:
This should actually be:

```
const decoder = new TextDecoder();
```

Reply:
Ah I see, even std uses `new TextDecoder()`. 👍